specs/spec.md

```diff
@@ -12,12 +12,12 @@
 1. [Abstract](#abstract)
 2. [Flow](#flow)
 3. [Wire Protocol](#wire-protocol)
-    - [Messages](#messages)
+    1. [Messages](#messages)
 4. [Transport](#transport)
 5. [Protocols](#protocols)
-    - [RPC](#rpc)
-    - [Gossip](#gossip)
-    - [PING](#ping)
+    1. [RPC](#rpc)
```

Comment:
protip: if you make all of these `1` then github will automatically number them for you in rendering. That way you don't need to go change all of the numbers if you add/remove in the middle of the list or reorder things.

Reply:
I use an editor that auto updates the numbers, so its like that when I copy over. Thanks for the tip tho @prestonvanloon